README.md

```diff
@@ -1,6 +1,11 @@
 
 # CODEU CHAT SERVER | README
 
+## UPDATE FROM YUHANG LIAO
```

Comment:
One other big issue - you've changed the file permissions of many files in this pull request. Notice that at the top, many files display a change like this:

![screen shot 2017-05-29 at 2 56 22 pm](https://cloud.githubusercontent.com/assets/1039411/26559668/03f9fde4-447f-11e7-8cfa-8e09f8edb2af.png)

That means these files have been changed to an "executable" status, which is incorrect - they are not executable files, and this can be confusing to other developers in the future. More importantly, a pull request should only change the files relevant to the feature being worked on. Before merging this, I strongly recommend that you change these files back to the default permissions. Assuming you're on Linux or Mac, for each of these files, you should run `chmod 0644 <file_name_here>` to revert them back to the correct status.

[This stackoverflow answer](https://stackoverflow.com/a/8347325/5053647) has more info about file permissions in case you're interested.